vignettes/split_functions.Rmd

```diff
@@ -291,6 +291,10 @@ The pre-processing component is useful for things such as manipulating
 factor levels, e.g., to trim unobserved ones or to reorder levels
 based on observed counts, etc.
 
+For further details on the structure of custom split functions, and an 
+example of a custom split function that does not implement `make_split_fun()`, 
```

Comment:
"does not implement" -> "not implemented via `make_split_fun()`

But a larger issue is I'm not sure we need/want this. I don't know of a single function that can't be done in `make_split_fun`...

Instead I'd say "for a more detailed discussion of what custom split functions do", or similar. And honestly, we likely need to merge `?custom_split_funs` into `?make_split_fun` at least partially, but we'll do that as a separate issue.